packages/trpc/server/createContext.ts

```diff
@@ -106,6 +106,7 @@ type CreateInnerContextOptions = {
   locale: string;
   user: Awaited<ReturnType<typeof getUserFromSession>>;
   i18n: Awaited<ReturnType<typeof serverSideTranslations>>;
+  ip: string;
```

Comment:
Add ip to identify the user

Reply:
If we can avoid this please do, what if we store a localhost reference to the booking in question? Reason being is that an ip is PII which means we'll need to do due diligence compliance wise.

Reply:
Can we use the next-auth cookie for this?

Reply:
yes please dont save the IP

Reply:
great catch @emrysal !